.github/workflows/release.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
       # client version: v2.15.2.
       - name: Install Helm
         run: |
-          curl -sSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get
+          curl -sSLo get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get
```

Comment:
this returns `404: Not Found`  to me

Reply:
Yeah, this instance will need to stay as Helm still uses the master branch

Reply:
🤦🏻‍♀️